static/app/views/insights/pages/platform/nextjs/index.tsx

```diff
@@ -34,16 +47,43 @@ export function NextJsOverviewPage({
       timestamp: date,
       version,
     })) ?? [];
+  const location = useLocation();
+  const navigate = useNavigate();
+  const queryParams = new URLSearchParams(location.search);
+
+  const activeView: View = (queryParams.get('view') as View) ?? 'api';
+  const spanOperationFilter: SpanOperation =
+    (queryParams.get('spanOp') as SpanOperation) ?? 'pageload';
```

Comment:
When using our wrappers the query object is available.

```suggestion
  const activeView: View = location.query.view ?? 'api';
  const spanOperationFilter: SpanOperation = location.query.spanOp ?? 'pageload';
```